incubator/exporter-otlp/src/main/java/io/aklivity/zilla/runtime/exporter/otlp/internal/duplicated/MetricRecord.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.exporter.otlp.internal.duplicated;
+
+public interface MetricRecord
+{
+    String namespaceName();
+
+    String bindingName();
+
+    String metricName();
```

Comment:
We don't need to suffix `Name` on these methods, they all return strings so it is already pretty clear.

Perhaps these should be ids though?

Reply:
The implementations (`CounterGaugeRecord` and `HistogramRecord`) get the id's in the constructor and resolve them to names (strings) for the very reason of making the names available for the exporters' purposes.

To fix your other comment I have just added the method `bindingId()`, so I think it's just better to keep the method names like `bindingName()`, `bindingId()` etc. just to make sure there is no ambiguity. What do you think? 🤔

Reply:
We put the binding name to Prometheus labels and Open Telemetry attributes, so we definitely need it.

Reply:
Right, but `EngineContext.supplyLocalName` already provides a lookup from ids to Strings that works for any `namespacedId` and `ExporterHandler` already has access to `EngineContext` so we only need the ids.

Lets have a design discussion on this to review the abstractions being introduced into the engine project for exporters.